3rdparty/CMakeLists.txt

```diff
@@ -152,4 +165,14 @@ if (IRR_BUILD_CEGUI)
 		DESTINATION ./debug/include/irr/ext/GUI
 		CONFIGURATIONS Debug
 	)
+	install(
+		FILES ${CEGUI_LIBS}
+		DESTINATION ./lib
```

Comment:
what does this do?

Reply:
It copies CEGUI's libraries to installation folder for end users to link.